.pre-commit-config.yaml

```diff
@@ -0,0 +1,13 @@
+repos:
+  - repo: https://github.com/psf/black
+    rev: 21.9b0
```

Comment:
Does this mean we have to keep it up to date? Not that it changes too much but still...

Anyway of NOT specifying a rev so latest is always used? Or is that more dangerous? Or major a major (`21`) only.

Anyway as I say, let's take it to another issue/PR.

Reply:
IIRC, you can specify "bleeding edge" if you always want the latest. This has the same downside as with packages, in that now each developer may be running with a different version, with different behavior.